### PR TITLE
feat(configuration): added support for unit suffix for maximum database size 

### LINKED ISSUE #315: Identical scan results are resent before unique-message-timeout
## Description

We configured Postee to not resend old scan results via the digest as unique-message-prop and a timeout of several days depending on the route. The problem is that this only works for a few hours, after which old results are resent again. The configured time period of several days is ignored.

## What did you expect to happen?
Resending of old results only after elapse of configured time period (unique-message-timeout).

## What happened instead?
It works for a certain period of time, i.e. when rescanning the images within a couple of hours after they were initially scanned, Postee recognizes old results and does not resend them again. But at some point, Postee seems to lose track of the old results. 
This is also reflected in the Bolt database: After scanning, key value pairs are written to "WebhookBucket" and "WebookExpiryDates". Looking at the database again after a few hours shows that all entries in "WebhookBucket" were deleted. "WebookExpiryDates" is still there with the correct timestamps x days from the time of scanning as per the unique-message-timeout.

## Output of run with `POSTEE_DEBUG=true`:

```
The logs don't show anything besides the usual result related messages. No log entry during the time period where deletion of "WebhookBucket" entries must have happened.
```

## Additional details (environment setup, networking info...):
Postee versions tried: 2.0.6 and 2.2.0
Openshift Dedicated on AWS, Postee storage on AWS EBS (gp2)

[posteeconfig_for_issue.txt](https://github.com/aquasecurity/postee/files/8212665/posteeconfig_for_issue.txt)

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
Added `<numbers><unit suffix>` pattern for `max-db-size` field in configuration file.
You can use a suffix: `b`, `kb`, `mb`, `gb`. Supports lowercase, uppercase and space between number and suffix. 
The value without the suffix is bytes. 
e.g.:
```yaml
max-db-size: 1000 # 1000b  
max-db-size: 1000b
max-db-size: 100000kb
max-db-size: 100 mb
max-db-size: 1 GB
```

## Issues

- Close #315

### CODE REVIEW

File: dbservice/checker.go [hunk 1]
```diff
@@ -33,7 +34,15 @@ func CheckSizeLimit() {
 			size += len(v)
 		}
 		if size > DbSizeLimit {
-			return tx.DeleteBucket([]byte(dbBucketName))
+			err = tx.DeleteBucket([]byte(dbBucketName))
```
Comment (AndreyLevchenko) [inline dbservice/checker.go:37]: do we have unit test for this logic?

Comment (DmitriyLewen) [reply dbservice/checker.go:37]: I changed some tests. We now have unit tests for the `CheckSizeLimit` and `DeleteBucket` functions. 
Can you take a look?

File: dbservice/checker_test.go [hunk 2]
```diff
@@ -53,45 +52,82 @@ func TestExpiredDates(t *testing.T) {
...
-		if test.needRun {
-			CheckSizeLimit()
-		}
+		t.Run(test.name, func(t *testing.T) {
+			DbSizeLimit = test.dbSizeLimit
+
+			db, err := bolt.Open(DbPath, 0666, nil)
+			if err != nil {
+				t.Fatal("Can't open db:", DbPath)
+				return
```
Comment (AndreyLevchenko) [inline dbservice/checker_test.go:88]: seems `return` is not required

Comment (DmitriyLewen) [reply dbservice/checker_test.go]: Thanks! Removed.

**Review by AndreyLevchenko — APPROVED** [review]

**Comment by simar7** [comment]
lgtm, could you rescan once more for the current usages as some new yaml's have been added since this PR was made. Once example is https://github.com/aquasecurity/postee/blob/main/cfg-docker-actions.yaml#L6

**Comment by DmitriyLewen** [comment]
Hello @simar7 
Thank you!
I pulled the changes and updated the new yaml.

**Comment by simar7** [comment]
> Hello @simar7
> Thank you!
> I pulled the changes and updated the new yaml.

awesome thanks!

**Pull request merged**